internal/utils/generate_scripts.go

```diff
@@ -0,0 +1,188 @@
+package utils
+
+import (
+	"os"
+	"text/template"
+
+	"github.com/NethermindEth/1Click/configs"
+	"github.com/NethermindEth/1Click/internal/templates"
+	log "github.com/sirupsen/logrus"
+)
+
+/*
+GenerateDockerComposeScripts :
+This function is responsible for generating docker-compose scripts for execution, consensus and
+validator clients.
+
+params :-
+a. executionClient string
+Execution client whose script is to be generated
+b. consensusClient string
+Execution client whose script is to be generated
+c. validatorClient string
+Execution client whose script is to be generated
+d. path string
+Path of generated scripts
+
+returns :-
+None
+*/
+func GenerateDockerComposeScripts(executionClient, consensusClient, validatorClient, path string) {
+	_, baseTmp := buildTemplate("docker-compose_base", "base")
+	_, executionTmp := buildTemplate(executionClient, "execution")
+	_, consensusTmp := buildTemplate(consensusClient, "consensus")
+	_, validatorTmp := buildTemplate(validatorClient, "validator")
+
+	tmp := template.Must(template.New("docker-compose").Parse(baseTmp))
```

Comment:
Bad idea to use .Must methods
.Must methods result in panics if some error occurs